src/extensions/yfm/GPT/hooks/useGpt.tsx

```diff
@@ -75,7 +75,10 @@ export const useGpt = <
                 setError(true);
             } finally {
                 setLoading(false);
-
+                setTimeout(() => {
+                    // rerender the popup
+                    window.dispatchEvent(new CustomEvent('scroll'));
+                });
```

Comment:
When a lot of text is entered into the GPT popup, it expands and goes beyond the boundaries. However, the popup does not handle height changes. So, you trigger it by hand. In an ideal solution, I expect that some method called like "recalculatePosition" will be called . Therefore, I suggest adding a comment about trigger scroll.